README.md

```diff
@@ -20,6 +20,8 @@ Requires [R](https://www.r-project.org/).
 
 ![Create R terminal](images/terminal.png)
 
+* Run code in terminal containing existing R session, for example over SSH (`Run Selection/Line in Active Terminal`)
+
```

Comment:
SSH support is awesome.
Do you have an example behavior of that?

Reply:
@Ikuyadeu I added a movie showing use of R in an SSH session to the README.